pkg/apiserver/utils/export.go

```diff
@@ -5,22 +5,108 @@ import (
 	"encoding/csv"
 	"errors"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"os"
-	"strings"
-
 	"github.com/Xeoncross/go-aesctr-with-hmac"
 	"github.com/gin-gonic/gin"
 	"github.com/gtank/cryptopasta"
-	"github.com/oleiade/reflections"
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
 
+	"github.com/oleiade/reflections"
 	"reflect"
 	"time"
 )
 
+const namespace = "export"
+
+func ExportCSV(data [][]string, filename string) (token string, err error) {
```

Comment:
So why not serving just from the memory?

Reply:
The current behavior is downloading immediately after exporting. The purpose of writing to the temp file should be to separate the behavior of exporting and downloading.
@baurine 
Should these two actions be combined (and save data in memory)? Or provide a new user action **share** that export and show user the link to download (seems to be meaningless for downloaded file can also be shared 😂)

Reply:
Tried to combine them and incorporate the `download` action into `query list`. It looks good but it's hard to save the data as a file due to the auto-generated request layer based on axios. Give up🤐